File: Solutions/reversewords.py
#Reverse Words in A Given String
#Problem Link: https://practice.geeksforgeeks.org/problems/reverse-words-in-a-given-string5459/1
# User function Template for python3

class Solution:
    
    #Function to reverse words in a given string.
    def reverseWords(self,S):
        array=S.split(".")
        total=len(array)
        S=""
        for i in range(total-1,0,-1):
           S+=array[i]
           S+="."
        S+=array[0]
        return S

File: Solutions/reversewords.java
public class reversewords {
    public static String reverseWord(String str){  
        String words[]=str.split("\\s");  
        String reversewords="";  
        for(String w:words){  
            StringBuilder sb=new StringBuilder(w);  
            sb.reverse();  
            reversewords+=sb.toString()+" ";  
        }  
        return reversewords.trim();  
    }  

    public class TestStringFormatter {  
        public static void main(String[] args) {  
            System.out.println(reversewords.reverseWord("my name is khan"));  
            System.out.println(reversewords.reverseWord("I am sonoo jaiswal"));    
            }  
        }  
}
